kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -533,6 +560,15 @@ public CompressionType getCompressionType() {
     return CompressionType.forName(getString(COMPRESSION_TYPE_CONFIG));
   }
 
+  public CompressionCodecName getCompressionCodecName() {
+    String parquetCompressionType = getString(PARQUET_COMPRESSION_TYPE_CONFIG);
+    if (parquetCompressionType.equals(PARQUET_COMPRESSION_TYPE_DEFAULT)) {
```

Comment:
hmm, this applies only to `none` right?
But if (for some reason) we changed the default value for this config then this would break. 
I'd suggest using specifically `"none"` here if this applies only to `"none"`